minos/api_gateway/rest/service.py

```diff
@@ -25,7 +28,12 @@ def __init__(self, address: str, port: int, config: MinosConfig):
         super().__init__(address, port)
 
     async def create_application(self) -> web.Application:
-        app = web.Application()
+        middlewares = []
+        if self.config.cors.enabled:
+            middlewares = [cors_middleware(allow_all=self.config.cors.enabled)]
```

Comment:
As you've already checked that the `self.config.cors.enabled` is `True`, it's preferable to directly pass the value (for readability reasons).
```suggestion
            middlewares = [cors_middleware(allow_all=True)]
```

Reply:
Done!